app/controllers/users_controller.rb

```diff
@@ -196,7 +196,8 @@ def activity
     @comments = Comment.joins(:comment_thread, :post).undeleted.where(user: @user, comment_threads: { deleted: false },
                                                                       posts: { deleted: false }).count
     @suggested_edits = SuggestedEdit.where(user: @user).count
-    @edits = PostHistory.joins(:post).where(user: @user, posts: { deleted: false }).count
+    @edits = PostHistory.joins(:post).where(user: @user, posts: { deleted: false },
+                                            post_history_type: PostHistoryType.find_by(name: 'post_edited')).count
```

Comment:
It'd be better to join the post history types table here and query that in the WHERE, rather than using a second query:

```suggestion
    @edits = PostHistory.joins(:post, :post_history_type).where(user: @user, posts: { deleted: false },
                                            post_history_types: { name: 'post_edited' }).count
```

Likewise for L212/213.